lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -188,7 +201,7 @@ class FeatureCard extends React.Component {
               <h2 className="heading -delta">{campaign.title}</h2>
               <p className="tagline">{campaign.tagline}</p>
               <ul className="form-actions">
-                {(campaign.signedUp && !this.state.isSignedUp) ? <h1>You're signed up already!</h1> : <li><button className={`${this.state.isSignedUp ? 'tada' : ''} button`} onClick={()=>this.signup()}>Sign Up</button></li>}
+                <li><Button loading={this.state.isLoading} success={this.state.isSignedUp} action={this.signup}></Button></li>
```

Comment:
And then here you can pass the bound function: `action={this.signup.bind(this)}`